### ISSUE TITLE
question about the Occlusion LineMOD dataset

### ISSUE THREAD

**Issue by trumDog** [ethnhe/PVN3D#63]
hello, thank you for your great work! 
I have a question about how to train the semantic segmentation branch on the Occlusion LineMOD dataset? Because it looks like this dataset has no ground truth semantic labels.

**Comment by ethnhe** [comment]
We trained the model on the LineMOD training set and only use the Occlusion LineMOD as the test set, following previous works.

**Comment by trumDog** [comment]
Got it, thanks!
